pdc/apps/auth/models.py

```diff
@@ -84,5 +84,5 @@ def email_user(self, subject, message, from_email=None, **kwargs):
         send_mail(subject, message, from_email, [self.email], **kwargs)
 
     def get_all_permissions(self, obj=None):
-        tmp_permission = _user_get_all_permissions(self, obj)
+        tmp_permission = PermissionsMixin.get_all_permissions(self, obj)
```

Comment:
Because PermissionsMixin already in User's class inheritance hierarchy, should it use `super` to access `get_all_permissions`? So if one day use other mixin, it will automatically use new mixin's `get_all_permissions`.

And again, these 2 lines may be replaced by one line.